lib/common/errors.js

```diff
@@ -0,0 +1,15 @@
+class FileNotExistsError extends Error {
+  constructor(message, status) {
+    super(message)
+
+    this.name = this.constructor.name
+
+    Error.captureStackTrace(this, this.constructor)
+
+    this.status = status || 500
```

Comment:
This seems copied from an error for HTTP. Can we simplify it?

Reply:
Solved!